runtime-testsuite/test/org/antlr/v4/test/runtime/PredictionMode.java

```diff
@@ -0,0 +1,7 @@
+package org.antlr.v4.test.runtime;
```

Comment:
Shouldn't we be retrieving this enum from the runtime ?

Reply:
No, it should not. This enum is used both for Java and Kotlin runtimes, they have its own `PredictionMode` declarations. It's not correctly to use Java runtime enum in Kotlin runtime tests.

Reply:
Not sure I follow, if it's not correct to use Java runtime enum in Kotlin runtime tests, how is it correct to use a generic test-suite enum in these Kotlin runtime tests ? (I'm genuinely trying to understand)

Reply:
We use [RuntimeTestDescriptor](https://github.com/antlr/antlr5/blob/dev/runtime-testsuite/test/org/antlr/v4/test/runtime/RuntimeTestDescriptor.java) as a descriptor that stores info about `.txt` and custom tests. It actually describes common data related to all runtimes, but not only Java-runtime. If we decide to completely drop Java-runtime, this enum will be unavailable and we have to drop its using anyway. I removed it within this PR to minimize future effort and to clarify code, see the commit https://github.com/KvanTTT/antlr4/pull/10/commits/e939f14092c786eaf48d9e27b5e8eff704a1383a

Reply:
Makes sense